integrationTests/multiShard/relayedTx/relayedTx_test.go

```diff
@@ -0,0 +1,440 @@
+package relayedTx
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm/arwen"
+	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/ElrondNetwork/elrond-go/process"
+	vmFactory "github.com/ElrondNetwork/elrond-go/process/factory"
+	"github.com/ElrondNetwork/elrond-go/process/smartContract/builtInFunctions"
+	"github.com/ElrondNetwork/elrond-go/vm/factory"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRelayedTransactionInMultiShardEnvironmentWithNormalTx(t *testing.T) {
+	if testing.Short() {
+		t.Skip("this is not a short test")
+	}
+
+	nodes, idxProposers, players, relayer, advertiser := createGeneralSetupForRelayTxTest()
+	defer func() {
+		_ = advertiser.Close()
+		for _, n := range nodes {
+			_ = n.Messenger.Close()
+		}
+	}()
+
+	sendValue := big.NewInt(5)
+	round := uint64(0)
+	nonce := uint64(0)
+	round = integrationTests.IncrementAndPrintRound(round)
+	nonce++
+
+	receiverAddress1 := []byte("12345678901234567890123456789012")
+	receiverAddress2 := []byte("12345678901234567890123456789011")
+
+	nrRoundsToTest := int64(5)
+	for i := int64(0); i < nrRoundsToTest; i++ {
+		for _, player := range players {
+			createAndSendRelayedAndUserTx(nodes, relayer, player, receiverAddress1, sendValue, integrationTests.MinTxGasLimit, []byte(""))
+			createAndSendRelayedAndUserTx(nodes, relayer, player, receiverAddress2, sendValue, integrationTests.MinTxGasLimit, []byte(""))
+		}
+
+		round, nonce = integrationTests.ProposeAndSyncOneBlock(t, nodes, idxProposers, round, nonce)
+		integrationTests.AddSelfNotarizedHeaderByMetachain(nodes)
+
+		time.Sleep(time.Second)
```

Comment:
(only if possible) - can we lower this sleep further (to shorten the test as much as possible)?